src/vdom.rs

```diff
@@ -141,7 +160,9 @@ where
     Mdl: 'static,
     ElC: View<Ms>,
 {
-    /// Stateless app configuration
+    /// Temporary app configuration that is removed after app begins running.
+    pub run_cfg: OptDynRunCfg<Ms, Mdl, ElC, GMs>,
```

Comment:
Could we rename `run_cfg` to something like `initial_cfg` or `temp_cfg`? (And `OptDynRunCfg` accordingly).

My intuition tells me that `run_cfg` is the "main" configuration which is available for the app lifetime, which is wrong.